docs/app demos/taker.md

```diff
@@ -128,12 +128,24 @@ The data directory contains the following files:
 
 The configuration is stored in the `config.toml` file. You can edit this file to change the configuration of the taker. The configuration file contains the following fields:
 
-1. `port` - The port via which the Taker listens and serves requests.
-2. `socks_port` - The port via which the Taker listens and serves requests for the Socks5 proxy.
-3. `rpc_port` - The port which serves the RPC server.
-4. `directory_server_address` - The address of the directory server.
-5. `connection_type` - The connection type to use for the directory server. Possible values are `CLEARNET` and `TOR`.
+- `network_port`:** Specifies the TCP port on which the Taker listens for incoming Coinswap protocol messages.  
+- `socks_port`:** Defines the SOCKS5 proxy port used to route network traffic through Tor.  
+- `directory_server_address:` Sets the Tor hidden service address of the Coinswap Directory Server for discovering Maker services.  
+- `connection_type:` The connection type to use for the directory server.(currently using `TOR`)
 
+
+---
+
+**Default Taker Configuration (`~/.coinswap/taker/config.toml`):**
+
+```toml
+network_port = 9051
```

Comment:
This should be `control_port`. This is a type in our code too. https://github.com/citadel-tech/coinswap/blob/9b74969edf79b5facd23a0d0621c52740302b054/src/taker/config.rs#L95

Can you fix it in this pr itself?